{"name": "test_basic_auth[--Not authorized\\n]", "status": "passed", "steps": [{"name": "Initialize URL.", "status": "passed", "parameters": [{"name": "username", "value": "''"}, {"name": "password", "value": "''"}], "start": 1761482137068, "stop": 1761482137068}, {"name": "Navigating to URL: 'http://the-internet.herokuapp.com/basic_auth'.", "status": "passed", "steps": [{"name": "Navigating to URL: 'http://the-internet.herokuapp.com/basic_auth'.", "status": "passed", "parameters": [{"name": "url", "value": "'http://the-internet.herokuapp.com/basic_auth'"}], "start": 1761482137069, "stop": 1761482137145}], "parameters": [{"name": "url", "value": "'http://the-internet.herokuapp.com/basic_auth'"}], "start": 1761482137068, "stop": 1761482137145}, {"name": "Get authorization message.", "status": "passed", "steps": [{"name": "Get text of element ('css selector', 'div#content p')", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x556be5caefea <unknown>\n#1 0x556be572cd06 <unknown>\n#2 0x556be577ee6d <unknown>\n#3 0x556be577f101 <unknown>\n#4 0x556be57cd6b4 <unknown>\n#5 0x556be57a4e3d <unknown>\n#6 0x556be57cab3b <unknown>\n#7 0x556be57a4be3 <unknown>\n#8 0x556be57712d2 <unknown>\n#9 0x556be5771f91 <unknown>\n#10 0x556be5c729e8 <unknown>\n#11 0x556be5c7684f <unknown>\n#12 0x556be5c59ec9 <unknown>\n#13 0x556be5c773f5 <unknown>\n#14 0x556be5c3f74f <unknown>\n#15 0x556be5c9bcc8 <unknown>\n#16 0x556be5c9bea3 <unknown>\n#17 0x556be5cadf83 <unknown>\n#18 0x7f7e1c89caa4 <unknown>\n#19 0x7f7e1c929c6c <unknown>\n\n", "trace": "  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/allure_commons/_allure.py\", line 202, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/selenium-python/selenium-python/pages/base_page.py\", line 113, in get_dynamic_element_text\n    elem = self.wait_for_visibility(locator, timeout)\n  File \"/home/runner/work/selenium-python/selenium-python/pages/base_page.py\", line 70, in wait_for_visibility\n    return self._safe_wait(EC.visibility_of_element_located, locator, timeout)\n  File \"/home/runner/work/selenium-python/selenium-python/pages/base_page.py\", line 28, in _safe_wait\n    return wait.until(ec_method(locator))\n  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "parameters": [{"name": "locator", "value": "('css selector', 'div#content p')"}, {"name": "timeout", "value": "None"}, {"name": "retry_count", "value": "2"}], "start": 1761482137145, "stop": 1761482147156}], "start": 1761482137145, "stop": 1761482147273}], "attachments": [{"name": "log", "source": "d08dc2ba-7ab2-450b-a79e-1cccf91d16b6-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "a48f93bd-5f2a-45e1-83ef-4141e26a8413-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "''"}, {"name": "password", "value": "''"}, {"name": "expected_message", "value": "'Not authorized\n'"}], "start": 1761482137068, "stop": 1761482147274, "uuid": "833b4ef5-a111-463d-a2e2-fb8fca4f1a57", "historyId": "42798e6512954325e5980ba5783125cc", "testCaseId": "1a21bc8c1f3aa66587e32fe54490333a", "fullName": "tests.test_basic_auth_login_scenarios.TestBasicAuth#test_basic_auth", "labels": [{"name": "severity", "value": "normal"}, {"name": "story"}, {"name": "feature", "value": "Basic Auth"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_basic_auth_login_scenarios"}, {"name": "subSuite", "value": "TestBasicAuth"}, {"name": "host", "value": "runnervmwhb2z"}, {"name": "thread", "value": "5668-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_basic_auth_login_scenarios"}], "titlePath": ["tests", "test_basic_auth_login_scenarios.py", "TestBasicAuth"]}